### PR TITLE
Update code-style.md

### PR DESCRIPTION
**Pull request by yvan-sraka**
Just use [eslint-config-meteor](https://github.com/dburles/eslint-config-meteor) as suggest in [this issue](https://github.com/meteor/guide/issues/581) (and fix `eslint` default path) :wink:!

### CODE REVIEW

**Comment by lorensr** [comment]
Thanks, waiting on a final npm publish location on that package

File: content/code-style.md [hunk 1]
```diff
@@ -99,25 +99,11 @@ You can also add a `eslintConfig` section to your `package.json` to specify that
 {
   ...
   "scripts": {
-    "lint": "eslint .",
+    "lint": "./node_modules/.bin/eslint .",
```
Comment (abernix) [inline content/code-style.md]: This shouldn't have to call `./node_modules/.bin` as that should be set automatically by the `npm` command when running `meteor npm run lint`.

Comment (yvan-sraka) [inline content/code-style.md]: You're right, I changed it!

**Comment by abernix** [comment]
It seems to me that we need to update the `meteor npm install` line a few lines up from this change now, test it, and merge it.  @yvan-sraka, is this something you have time to help with?

**Comment by yvan-sraka** [comment]
@abernix, I just did it and I tested it, everything seems good!

**Comment by abernix** [comment]
LGTM, thanks!.  We might want to revisit this based on the outcome of https://github.com/meteor/eslint-config-meteor/pull/3, but let's land this for now.

**Pull request merged**